src/ws/index.ts

```diff
@@ -0,0 +1,18 @@
+import { QueryClientConfig } from '../types';
+import { WebsocketClient, configureWebsocketClient } from './ws-client';
+
+/**
+ * Graasp websocket client top-level file
+ * Entry point to use the Graasp WebSocket client in front-end applications
+ */
+export * from './hooks';
+export { configureWebsocketClient } from './ws-client';
+export { default as MockWebSocket } from './mock-ws-client';
+
+export const getWebsocketClient = (queryConfig: QueryClientConfig): WebsocketClient | undefined => {
+  const { enableWebsocket, WS_HOST } = queryConfig;
+  if (enableWebsocket && typeof WS_HOST !== 'undefined') {
+    return configureWebsocketClient({ ...queryConfig, WS_HOST });
+  }
+  return;
+};
```

Comment:
Not a fan of putting this in production code. Can you use something similar to `jest.mock` in your test cases instead?

Reply:
I am not sure what the issue is here ? Is it the check on `typeof WS_HOST !== "undefined"` ?